otx/algorithms/common/adapters/mmcv/models/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""Adapters for OTX Common Algorithm. - mmcls.model."""
+"""Adapters for OTX Common Algorithm. - mmcv.model."""
```

Comment:
Is this for only mmcv model? I can see the torchvision models

Reply:
Thanks for the review!
Register the torchvision model in the mmcv registry and use it. In addition, OTX's custom backbones are registered and used in the mmcv registry, so these are mmcv models.